tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
1) TendrlContext is save to etcd by Import/Create flows
2) To load() TendrlContext from etcd you need only NS.node_context.node_id

3) Put this fix for ceph-integration, gluster-integration , performance_mon, alerting (cc: @anmolbabu )
https://github.com/Tendrl/node-agent/commit/15dee0f2280a8290a6d25f4a8d5425ef79750b43

4) Above fix basically loads the TendrlContext if any (from etcd) at Tendrl service startup 

This way we dont need a disk local TendrlContext 

Let me know we need to debug this more